### PR TITLE
Extend German translation

### PR DESCRIPTION
**Pull request by wlhlm**
Adding a few bits that were missing.

### CODE REVIEW

File: src/main/resources/assets/openblocks/lang/de_DE.lang [hunk 1]
```diff
@@ -70,13 +86,24 @@ openblocks.misc.selected_cannon=Kanone zum Zielen ausgewählt
...
+openblocks.misc.change_box_size=Ändere Größe zu (%d,%d,%d):(%d,%d,%d)
 openblocks.misc.total_blocks=Insgesamt gezählte Blöcke: %d
 openblocks.misc.get_witched=Werde verhext!
-
-openblocks.misc.cant_restore=Das Inventar von Spieler %s kann nicht wiederhergestellt werden
-openblocks.misc.cant_store=Das Inventar von Spieler %s kann nicht gespeichert werden
-openblocks.misc.stored_inventory=Inventar des Spielers gespeichert in %s
+openblocks.misc.page=Seite %d von %d
+openblocks.misc.oh_no_ceiling=Du kannst hier nicht schlafen. Die Decke beunruhigt dich zu sehr...
+openblocks.misc.oh_no_ground=Auf dem DING willst du schlafen?!
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_DE.lang]: How about having it "Auf DEM Ding willst du schlafen ?!" Sounds a bit better, in my opinion.

Comment (wlhlm) [inline src/main/resources/assets/openblocks/lang/de_DE.lang]: Yeah, makes more sense.

**Comment by wlhlm** [comment]
Updated based on feedback.

File: src/main/resources/assets/openblocks/lang/de_DE.lang [hunk 2]
```diff
@@ -24,20 +35,25 @@ openblocks.gui.itemdropper=Itemspender
 openblocks.gui.blockplacer=Blockplatzierer
 openblocks.gui.paintmixer=Farbmixer
 openblocks.gui.drawingtable=Zeichentisch
-openblocks.gui.blocks=Blöcke
-openblocks.gui.items=Items
-openblocks.gui.misc=Misc
 openblocks.gui.digitalfuse=Digitalzünder
+openblocks.gui.xp_outputs=XP-Abgabe
+openblocks.gui.item_outputs=Itemabgabe
+openblocks.gui.watch_video=Sehe Video
```
Comment (XFactHD) [inline src/main/resources/assets/openblocks/lang/de_DE.lang]: Should be "Siehe Video".

File: src/main/resources/assets/openblocks/lang/de_DE.lang [hunk 3]
```diff
@@ -24,20 +35,25 @@ openblocks.gui.itemdropper=Itemspender
...
 
+openblocks.gui.restore_inv.title=Inventar-Backup
+openblocks.gui.restore_inv.content=Jeder kann einen Fehler machen, aber mit dem §L/ob_inventory§R-Befehl kannst du einige von ihnen wieder beheben. Dieser Befehl kann zum Wiederherstellen des Inventars genutzr werden. Ein Backup wird erstellt wenn:\n - der Spieler stirbt\n - ein Grabstein gespawnt wird \n - der Spieler den Befehl §L/ob_inventory store§R eingibt\n\nTo restore an inventory you need the name of the backup. Gehen in den unten genannten Ordner oder den saves-Ordner bei Servern und such nach Dateien mit dem Namen §Linventory-*.dat§R. Dann stelle das Inventar wieder her mit §L/ob_inventory restore <player> <id>§R - wobei id der volle Dateiname ist, oder nur der mittlere Teil ohne §Linventory-§R and §L.dat§R. Du kannst Tabvervollständigung benutzen. Backups sind normal NBT-Dateien und können mit einem entsprechenden Editor geöffnet werden.
```
Comment (XFactHD) [inline src/main/resources/assets/openblocks/lang/de_DE.lang]: Should be "genutzt".

File: src/main/resources/assets/openblocks/lang/de_DE.lang [hunk 4]
```diff
@@ -70,13 +86,24 @@ openblocks.misc.selected_cannon=Kanone zum Zielen ausgewählt
...
+
+openblocks.misc.grave_msg=%s (Tag: %.1f)
+
+openblocks.misc.cant_restore_player=Kann Inventar für %s nicht wiederherstellen
+openblocks.misc.cant_restore_inventory=Kann Inventar nicht wiederherstellen
+openblocks.misc.cant_store=Das Inventar für %s kann nicht gespeichert werden
+openblocks.misc.invalid_sub_inventory=Kann Unterinventar %s nicht widerherstellen
+openblocks.misc.stored_inventory=Spielerinventar gespeichert in %s
 openblocks.misc.restored_inventory=Inventar von Spieler %s wiederhergestellt
+openblocks.misc.invalid_index=Invalider index
```
Comment (XFactHD) [inline src/main/resources/assets/openblocks/lang/de_DE.lang]: Should be "Fehlerhafter Index".

File: src/main/resources/assets/openblocks/lang/de_DE.lang [hunk 5]
```diff
@@ -135,18 +171,29 @@ openblocks.turtle.magnet=Magnet
...
 
 tile.openblocks.guide.name=Bau-Hilfe
 tile.openblocks.guide.description=Die Bau-Hilfe wird dir eine Schablone aus unechten Blöcken geben, in unterschiedlichen Formen und Größen, um dir zu helfen, deine Bauten zu planen.\nDu kannst den Block Shift-rechtsklicken, um die Form zu verändern, oder du kannst verschiedene Seiten anklicken, um die Schablone zu vergößern bzw. zu verkleinern.\nIm Kreativ-Modus kannst du einen Obsidian-Block oberhalb der Bau-Hilfe platzieren und diese dann mit irgendeinem Block rechtsklicken, um die Schablone automatisch mit diesem Block auszufüllen.
+tile.openblocks.guide.video=http://www.youtube.com/watch?v=IxSZ-mOFHX0
+
+tile.openblocks.builder_guide.name=Verbesserte Bau-Hilfe
+tile.openblocks.builder_guide.description=Die verbesserte Form der Bau-Hilfe zeigt nicht nur Geist-Blöcken zum einfacheren Bauen, sondern erlaubt auch das Setzen der eigentlichen Blöcke. Klicke dazu einfach auf den zentralen Block mit einem beliebigen Stapl!\nIm Kreativmodus kannst du einen Obisidanblock darüber platzieren und dann die Bau-Gilfe mit einem beliebigen Block hauen um das Gebäude zu bauen.
```
Comment (XFactHD) [inline src/main/resources/assets/openblocks/lang/de_DE.lang]: Should be "Phantomblöcke", "Stapel" and "Bau-Hilfe".

File: src/main/resources/assets/openblocks/lang/de_DE.lang [hunk 6]
```diff
@@ -135,18 +171,29 @@ openblocks.turtle.magnet=Magnet
...
+tile.openblocks.builder_guide.video=http://www.youtube.com/watch?v=jymwA3X3ZtY
 
-tile.openblocks.ladder.name=Jaded-Leiter
-tile.openblocks.ladder.description=Die Jaded-Leiter löst das nervige Problem, über eine Falltür an der Spitze einer Leiter springen zu müssen. Die Jaded-Leiter funktioniert zeitgleich wie eine Falltür und wie eine Leiter.
+tile.openblocks.ladder.name=Erschöpfte Leiter
+tile.openblocks.ladder.description=Die Erschöpfter Leiter löst das nervige Problem, über eine Falltür an der Spitze einer Leiter springen zu müssen. Die Erschöpfte Leiter funktioniert zeitgleich wie eine Falltür und wie eine Leiter.
```
Comment (XFactHD) [inline src/main/resources/assets/openblocks/lang/de_DE.lang]: Should be "Strickleiter" instead of "Erschöpfte Leiter"

**Comment by wlhlm** [comment]
Updated.

Thanks for the feedback!

**Comment by wlhlm** [comment]
Updated again.

Removed the video links as per 98f41070a6f159d94695c3ea7b8fb8cb29984f88.

Please let me know what else is holdings this back from merging.

**Comment by boq** [comment]
Nothing. I just wait for release to merge it, to limit amount of PRs.

**Comment by wlhlm** [comment]
I see. I saw the other open translation PRs and was a little worried.

**Pull request merged**